docs/330-contributing.md

```diff
@@ -0,0 +1,37 @@
+3. Writing tests. Nobody loves it, everybody needs it, and we love if you do it!
+4. Documentation. Both our [technical manual](https://openvoiceos.github.io/ovos-technical-manual/) and our [community docs](https://openvoiceos.github.io/community-docs/) are community-managed and could use PRs ranging from grammatical fixes, fixing broken links, and clarifying/expanding on existing information. #goodfirstissue
+5. Testing out skills and looking for flaws/ways to improve. Many of our skills have been released and in the wild for years, but could use some loving attention. You can find actively supported skills in the `OpenVoiceOS` org, or skills in need of some care in the `OVOSHatchery` org.
+
+## How do you get started?
+
+First of all, if you're just starting out and want to contribute documentation or localization PRs, don't sweat it if you don't have OVOS running. We're happy to help with testing those. If you'd like to run it yourself and confirm things work as you expect, read on.
+
+OVOS has a [reference setup for Docker](https://github.com/OpenVoiceOS/ovos-docker) that will run on any platform and architecture that uses Docker - Linux, Mac OS, and Windows with WSL2. If you're comfortable with Docker, this is the fastest and easiest way to get OVOS running locally to contribute back. (Yes, it also works with Podman!)
+
+If you have a spare Raspberry Pi with a microphone/speaker available, OVOS maintains [a ready-to-use image](https://openvoiceos.github.io/community-docs/qs_intro/#rasberry-pi-os-latest-images) for that platform. It is currently not fully tested on the Pi 5.
+
+Raspberry Pi also supports GUI skills (that means if you have a screen attached to your Pi, you can run skills that have a graphical interface), which are written in Qt. If you're a Qt developer, we'd love to have your help!
+
+The easiest way to get OVOS running on any given Linux system is to use [ovos-installer](https://github.com/openvoiceos/ovos-installer).
+
+The most up-to-date information on running OVOS can be found [in our community docs](https://openvoiceos.github.io/community-docs/qs_intro/).
+
+## Skills and plugins
+
+The docs above are meant for a broad set of users. For the details you're looking for on how all these different pieces of OVOS interact, [please check out our technical manual](https://openvoiceos.github.io/ovos-technical-manual/). 
+
+A friend-of-OVOS, [Neon.AI](https://neon.ai), has a comprehensive set of [documentation for skill development](https://neongeckocom.github.io/neon-docs/skill_development/voice-user-interface-design-guidelines/what-can-a-skill-do/) specifically. Much of this was forked from the (excellent) Mycroft skill documentation and covers diverse topics such as design philosophy, available helper methods, writing GUIs (Qt devs wanted!), how to get a skill to converse back and forth with a user, and more.
```

Comment:
we have our own dev docs in the technical manual, they should include anything from old mycroft docs that was salvageable, don't think we should link Neon for info about our skills? if theres something in Neon docs we are missing then a PR should be sent to technical manual

Reply:
thanks to bringing this in notice